components/CredentialDetailsTree.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="q-pa-md q-gutter-sm">
+    <q-tree
```

Comment:
I imagine this doesn't handle deeply nested data very well (probably uses a lot of screen width), but maybe that's not true? So this is probably a good start for this feature, but my guess is that we'll need to do more.

We also want to be careful with how we expose this kind of information to users. Can we mark it somehow as a developer-only view? I would expect users will read the natural language names of the properties and presume those match their meaning -- when this isn't necessarily accurate.

Reply:
> ...doesn't handle deeply nested data
The most nested example credentials are 5 levels deep which render well. To your point, there is a limit, hence this being more of a useful tool than a permanent solution. I thought this would be a good first iteration, especially since the tree component is a Quasar offering. Long term, we could look at building out your idea of color coding rather than indenting related fields.

> how we expose this 
I can definitely hide it for development only. I think this is very useful, even to the common user, but the naming conventions will certainly be confusing. Again, agreeing with your point, I think this request pops up for developers more than the common future user.

Reply:
Resolved in 546ae33